cover.exs

```diff
@@ -0,0 +1,29 @@
+#!bin/elixir
+
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2021 The Elixir Team
+
+# TODO: Move this file in an acceptable location
+
+cover_dir = __ENV__.file |> Path.dirname() |> Path.join("cover")
+coverdata_inputs = cover_dir |> Path.join("{exunit,eunit}_*.coverdata") |> Path.wildcard()
+coverdata_output = Path.join(cover_dir, "combined.coverdata")
+ebins = __ENV__.file |> Path.dirname() |> Path.join("lib/*/ebin") |> Path.wildcard()
+
+_ = :cover.stop()
+{:ok, _pid} = :cover.start()
+
+for ebin <- ebins do
+  # TODO: Check result
+  :cover.compile_beam_directory(String.to_charlist(ebin))
+end
+
+for file <- coverdata_inputs do
+  :ok = :cover.import(String.to_charlist(file))
+end
+
+:ok = :cover.export(String.to_charlist(coverdata_output))
+
+{:ok, _} = Application.ensure_all_started(:mix)
+# TODO: How can I mute the "Analysis includes data from imported files" IO?
```

Comment:
I wouldn't mind leaving it as is :)

Reply:
@josevalim It's very hard to find the actual coverage information since there's a lot of IO. I had to increase my terminal buffer size to even find the info I was looking for.

Reply:
I see, I found this code in test.coverage.ex:

```
        # Silence analyse import messages emitted by cover
        {:ok, string_io} = StringIO.open("")
        Process.group_leader(pid, string_io)
        Mix.shell().info("")
        generate_cover_results(test_coverage)
```